master/buildbot/test/unit/test_status_web_base.py

```diff
@@ -61,6 +61,14 @@ def check(f):
 
 class Functions(unittest.TestCase):
 
+    def setUp(self):
+        from buildbot.config import MasterConfig
```

Comment:
I am not sure that we need it. It is a test and creating a big object for two properties is the triumph of form over content. Create this fields with constant value, not from the `MasterConfig` object. It is a test, we do not need any dependencies within

Reply:
I moved these tests to the new file in integration/ folder. I changed MasterConfig to constant variables.